hack/run_with_local_webhook.sh

```diff
@@ -290,6 +290,17 @@ if [ -n "${CSV_NAME}" ]; then
 
     oc patch "${CSV_NAME}" -n openstack-operators --type=json -p="[{'op': 'replace', 'path': '/spec/install/spec/deployments/0/spec/replicas', 'value': 0}]"
     oc patch "${CSV_NAME}" -n openstack-operators --type=json -p="[{'op': 'replace', 'path': '/spec/webhookdefinitions', 'value': []}]"
+else
+    # Handle operator deployed by Openstack Initialization resource
+    CSV_NAME="$(oc get csv -n openstack-operators -l operators.coreos.com/openstack-operator.openstack-operators -o name)"
```

Comment:
Watcher-operator is a standalone operator. It will be integrated with openstack-operator with FR4 timeframe. 
Is there anything we can modify here to work with existing case and add openstack-operator case in future?

Reply:
Good catch, Ok then can be added once it's integrated, closing for now.